zepben/ewb-sdk-jvm
# [DEV-1534] `Switch.ratedCurrent` and `SynchronousMachine.reactiveCapabilityCurveMRIDs` fixes

# Description

Made the following changes:
* Changed `Switch.ratedCurrent` to a float to support milli-amp ratings of fuses.
* `SynchronousMachine.reactiveCapabilityCurveMRIDs` is now a list of strings. This is what it should have always been (missed in the PR review of the initial change).
* Added docstrings.

# Associated tasks

- https://github.com/zepben/ewb-grpc/pull/111
- https://github.com/zepben/ewb-network-routes/pull/149
- https://github.com/zepben/network-verifier-lib/pull/47

# Test Steps

1. Update a database and make sure the sqlite file has updated the `rated_current` column of all switch tables to be a float, and your values are still there.
2. Request a `SynchronousMachine` via gRPC and ensure the curves collection is a list of mRID's rather than the curves themselves.

or 

rely on the extensive checks in the unit tests that cover this for you.

# Checklist

If any of these are not applicable, strikethrough the line `~like this~`. **Do not delete it!**. Let the reviewer decide if you should have done it.

### Code

### Documentation
~- [ ] I have updated any documentation required for these changes.~

# Breaking Changes

Both changes are breaking.
